package.json

```diff
@@ -1,5 +1,15 @@
 {
+    "type": "module",
     "dependencies": {
-      "cosmiconfig": "8.0.0"
+        "@commitlint/config-conventional": "^19.1.0",
+        "@commitlint/types": "^19.0.3",
```

Comment:
@webwarrior-ws by inserting these 2 lines above, you're contradicting what the file `version.config` says